### PR TITLE
enable extended test for old config

### PR DESCRIPTION
**Pull request by deads2k**
Add the config compatibility test to our extended bucket.  This ensures that we can always run openshift with an old config.

@liggitt ptal for the compatibility concerns

@mfojtik concerns about launching?

### CODE REVIEW

**Comment by deads2k** [comment]
[test] [extended:config-compatibility]

File: test/old-start-configs/v1.0.0/test-end-to-end.sh [hunk 1]
```diff
@@ -242,6 +242,13 @@ wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.2
 wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 80
 wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/api/v1beta3/nodes/${KUBELET_HOST}" "apiserver(nodes): " 0.25 80
 
+# COMPATBILITY update the cluster roles so that new images can be used.
+oadm policy reconcile-cluster-roles --confirm
```
Comment (liggitt) [inline test/old-start-configs/v1.0.0/test-end-to-end.sh]: this is fine... I'd expect to have to run this between versions

File: test/old-start-configs/v1.0.0/test-end-to-end.sh [hunk 2]
```diff
@@ -256,7 +263,8 @@ echo "Log in as 'e2e-user' to see the 'test' project."
 
 # install the router
 echo "[INFO] Installing the router"
-openshift admin router --create --credentials="${MASTER_CONFIG_DIR}/openshift-router.kubeconfig" --images="${USE_IMAGES}"
+# COMPATBILITY add --service-account parameter
```
Comment (liggitt) [inline test/old-start-configs/v1.0.0/test-end-to-end.sh]: s/COMPATBILITY/COMPATIBILITY/g

**Comment by deads2k** [comment]
typos fixed.  @liggitt anything else?

**Comment by liggitt** [comment]
Not really... the updates seem reasonable... it would be nice to have a running list of changes needed to keep a script working with the current version, but I guess searching for COMPATIBILITY in the script isn't that hard

**Comment by deads2k** [comment]
> it would be nice to have a running list of changes needed to keep a script working with the current version, but I guess searching for COMPATIBILITY in the script isn't that hard

I debated, but ultimately decided we were better off doing a grep in the actual running file.

[merge]

**Pull request merged**